jaxrs-spec/src/main/asciidoc/chapters/context/_contexttypes.adoc

```diff
@@ -21,26 +21,26 @@ other types are server-side only.
 ==== Application
 
 The instance of the application-supplied `Application` subclass can be
-injected into a class field or method parameter using the
-`@Context` annotation. Access to the `Application` subclass instance
-allows configuration information to be centralized in that class. Note
-that this cannot be injected into the `Application` subclass itself
-since this would create a circular dependency.
+injected a method parameter, since it is a CDI bean, or injected into
```

Comment:
"injected as a method parameter"?

Reply:
I think we should say "resource method parameter" not just "method parameter" given that an arbitrary method that isn't a resource method, won't support that. I guess this comment applies to other cases below.

Reply:
I agree and have applied the recommended changes.

Reply:
One additional comment, the switch to CDI also allows for injection into constructors, which didn't work all the time before, and that's why it was not mentioned explicitly in this section.